errors.js

```diff
@@ -0,0 +1,28 @@
+/* eslint no-undefined: 0 */
+'use strict';
+
+const format = require('./format');
+const { MESSAGE } = require('triple-beam');
+
+/*
+ * function simple (info)
+ * Returns a new instance of the simple format TransformStream
+ * which writes a simple representation of logs.
+ *
+ *    const { level, message, splat, ...rest } = info;
+ *
+ *    ${level}: ${message}                            if rest is empty
+ *    ${level}: ${message} ${JSON.stringify(rest)}    otherwise
+ */
```

Comment:
The docs here are wrong (copied from simple.js). 

...will change upon approval of the code

Reply:
I agree, this is a nice way to add error support.  Can we get the docs updated here and then we can finally hit merge? : )